packages/lesswrong/server/callbacks/votingCallbacks.ts

```diff
@@ -30,10 +30,11 @@ export async function onVoteCancel(newDocument: DbVoteableType, vote: DbVote, co
   void revokeUserAFKarmaForCancelledVote({newDocument, vote});
   
   
+  // TODO: Remove if we stop using contributorScores
```

Comment:
I don't think we need this TODO if the EA forum is using them (or the one on L49)